webviz_subsurface/datainput/_intersect.py

```diff
@@ -0,0 +1,34 @@
+import os
+import pandas as pd
+try:
+    import xtgeo
+except ImportError:
+    pass
+
+
+def load_well(well_name):
+    return xtgeo.well.Well(well_name)
+
+
+def load_surface(s_name, real_path, surface_cat):
+    path = os.path.join(real_path, 'share/results/maps',)
+    s_path = f'{s_name}--{surface_cat}.gri'
+    s_path = os.path.join(path, s_path)
```

Comment:
Combine line 15 and 16?
`s_path = os.path.join(path, f'{s_name}--{surface_cat}.gri')`